best-practices/README.md

```diff
@@ -220,6 +220,11 @@ Browsers
 
 * Don't support versions of Internet Explorer before IE9.
 
+DNS
+---
+
+* Avoid using non-standard DNS entries (i.e. ALIAS, ANAME).
```

Comment:
Super grammatical nitpick but this feels more like an e.g. than an i.e. to me ("for example" instead of "that is").